--- pciutils-2.1.8/pci.ids.ami	Tue Dec 12 11:46:22 2000
+++ pciutils-2.1.8/pci.ids	Tue Dec 12 11:47:50 2000
@@ -432,6 +432,9 @@
 	9712  Pipeline 9712
 	c24a  90C
 101e  American Megatrends Inc.
+	1960  MegaRAID
+		1028 0471 PowerEdge RAID Controller 3/QC
+		1028 0493 PowerEdge RAID Controller 3/DC
 	9010  MegaRAID
 	9030  EIDE Controller
 	9031  EIDE Controller
@@ -1108,6 +1111,8 @@
 	1022  ISP1022
 	1080  QLA1080
 	1216  QLA12160
+		101e 8471 QLA12160 on AMI MegaRAID
+		101e 8493 QLA12160 on AMI MegaRAID
 	1240  QLA1240
 	1280  QLA1280
 	2020  ISP2020A
