--- pciutils-2.1.8/pci.ids.i860	Tue Feb 20 14:44:57 2001
+++ pciutils-2.1.8/pci.ids	Tue Feb 20 18:09:51 2001
@@ -4628,7 +4628,9 @@
 	2520  82805AA MTH Memory Translator Hub
 	2521  82804AA MRH-S Memory Repeater Hub for SDRAM
 	2530  82850 850 (Tehama) Chipset Host Bridge (MCH)
+	2531  82850 860 (Wombat) Chipset Host Bridge (MCH)
 	2532  82850 850 (Tehama) Chipset AGP Bridge
+	2533  82860 860 (Wombat) Chipset AGP Bridge
 	5200  EtherExpress PRO/100
 	5201  EtherExpress PRO/100
 		8086 0001  EtherExpress PRO/100 Server Ethernet Adapter
