diff -Nur pciutils-2.2.5.orig/lib/configure pciutils-2.2.5.LDFLAGS/lib/configure
--- pciutils-2.2.5.orig/lib/configure	2007-02-06 11:58:14.000000000 +0000
+++ pciutils-2.2.5.LDFLAGS/lib/configure	2007-05-17 21:20:01.000000000 +0000
@@ -92,7 +92,7 @@
 		echo >>$c '#define PCI_HAVE_PM_NBSD_LIBPCI'
 		echo >>$c '#define PCI_PATH_NBSD_DEVICE "/dev/pci0"'
 		echo >>$m 'PCILIB=lib/libpciutils.a'
-		echo >>$m 'LDFLAGS+=-lpci'
+		echo >>$m 'LIBS+=-lpci'
 		;;
     	gnu)
 		echo_n " i386-ports"
@@ -121,7 +121,7 @@
 if [ "$zlib" = yes ] ; then
 	echo >>$c '#define PCI_COMPRESSED_IDS'
 	echo >>$c '#define PCI_IDS "pci.ids.gz"'
-	echo >>$m 'LDFLAGS+=-lz'
+	echo >>$m 'LIBS+=-lz'
 else
 	echo >>$c '#define PCI_IDS "pci.ids"'
 fi
diff -Nur pciutils-2.2.5.orig/Makefile pciutils-2.2.5.LDFLAGS/Makefile
--- pciutils-2.2.5.orig/Makefile	2007-05-17 21:18:52.000000000 +0000
+++ pciutils-2.2.5.LDFLAGS/Makefile	2007-05-17 21:20:37.000000000 +0000
@@ -38,8 +38,13 @@
 	cd pci && ./configure $(IDSDIR) $(VERSION) $(HOST) $(RELEASE)
 
 pcimodules: pcimodules.o common.o $(PCILIB)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^ $(LIBS)
+
 lspci: lspci.o common.o $(PCILIB)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^ $(LIBS)
+
 setpci: setpci.o common.o $(PCILIB)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^ $(LIBS)
 
 lspci.o: lspci.c pciutils.h $(PCIINC)
 setpci.o: setpci.c pciutils.h $(PCIINC)
