--- pciutils-3.10.0/lib/init.c.orig	2023-05-04 21:14:07.869516839 +0200
+++ pciutils-3.10.0/lib/init.c	2023-05-04 21:29:51.238144530 +0200
@@ -440,8 +440,10 @@
 
   if (a->method != PCI_ACCESS_AUTO)
     {
-      if (a->method >= PCI_ACCESS_MAX || !pci_methods[a->method])
+      if (a->method >= PCI_ACCESS_MAX || !pci_methods[a->method]) {
 	a->error("This access method is not supported.");
+	return 0;
+      }
       a->methods = pci_methods[a->method];
     }
   else
