--- pciutils-2.1.8/pci.ids.org	Mon Feb 19 16:07:40 2001
+++ pciutils-2.1.8/pci.ids	Mon Feb 19 16:08:45 2001
@@ -40,6 +40,8 @@
 	a0f0  Advanced System Management Controller
 	a0f3  Triflex PCI to ISA Bridge
 	a0f7  PCI Hotplug Controller
+		8086 002a  PCI Hotplug Controller A
+		8086 002b  PCI Hotplug Controller B
 	a0f8  USB Open Host Controller
 	ae10  Smart-2/P RAID Controller
 		0e11 4030  Smart-2/P Array Controller
@@ -1652,6 +1654,8 @@
 		10f7 8312  MagicGraph 128XD
 	0005  [MagicMedia 256AV]
 	0006  NM2360 [MagicMedia 256ZX]
+	0016  NM2380 [MagicMedia 256XL+]
+		10c8 0016  MagicMedia 256XL+
 	0083  [MagicGraph 128ZV Plus]
 	8005  [MagicMedia 256AV Audio]
 		0e11 b0d1  MagicMedia 256AV Audio Device on Discovery
@@ -2401,6 +2405,7 @@
 118b  Hypertec Pty Limited
 118c  Corollary, Inc
 	0014  PCIB [C-bus II to PCI bus host bridge chip]
+	1117  Intel 8-way XEON Profusion Chipset [Cache Coherency Filter] 
 118d  BitFlow Inc
 	0001  Raptor-PCI framegrabber
 	0012  Model 12 Road Runner Frame Grabber
@@ -3927,6 +3932,8 @@
 		14f1 2015  SoftK56 Speakerphone Winmodem
 	2016  SoftK56 Speakerphone Winmodem
 		14f1 2016  SoftK56 Speakerphone Winmodem
+	2443  SoftK56 Speakerphone Winmodem
+		14f1 2443  SoftK56 Speakerphone Winmodem
 14f2  MOBILITY Electronics
 14f3  BROADLOGIC
 14f4  TOKYO Electronic Industry CO Ltd
@@ -4492,6 +4499,7 @@
 	04a3  82434LX [Mercury/Neptune]
 	04d0  82437FX [Triton FX]
 	0960  80960RP [i960 RP Microprocessor/Bridge]
+	0964  80960RP [i960 RP Microprocessor/Bridge]
 	1000  82542 Gigabit Ethernet Adapter
 		0e11 b0df  NC1632 Gigabit Ethernet Adapter
 		0e11 b0e0  NC1633 Gigabit Ethernet Adapter
@@ -4518,9 +4526,10 @@
 		0e11 b0c6  Embedded NC3120 with Wake on LAN
 		0e11 b0c7  Embedded NC3121
 		0e11 b0d7  NC3121 with Wake on LAN
-		0e11 b0dd  NC3131
+		0e11 b0dd  NC3131 (82558B)
 		0e11 b0de  NC3132
 		0e11 b0e1  NC3133
+		0e11 b144  NC3123 (82559)
 		1014 005c  82558B Ethernet Pro 10/100
 		1014 105c  Netfinity 10/100
 		1033 8000  PC-9821X-B06
@@ -4575,6 +4584,8 @@
 	1960  80960RP [i960RP Microprocessor]
 		101e 0438  MegaRaid 438
 		101e 0466  MegaRaid 466
+		101e 0467  MegaRaid 467
+		1028 0467  PowerEdge Expandable RAID Controller 2/DC
 		103c 10c6  MegaRaid 438
 		103c 10c7  MegaRaid T5
 		1111 1111  MegaRaid 466
