--- pciutils-2.2.9/update-pciids.sh~	2006-09-09 13:36:17.000000000 +0300
+++ pciutils-2.2.9/update-pciids.sh	2008-04-15 03:40:37.506541329 +0300
@@ -19,23 +19,23 @@
 	DECOMP="cat"
 	SRC="$SRC.gz"
 	GREP=zgrep
-elif which bzip2 >/dev/null 2>&1 ; then
+elif [ -x /usr/bin/bzip2 ]; then
 	DECOMP="bzip2 -d"
 	SRC="$SRC.bz2"
-elif which gzip >/dev/null 2>&1 ; then
+elif [ -x /usr/bin/gzip ]; then
 	DECOMP="gzip -d"
 	SRC="$SRC.gz"
 else
 	DECOMP="cat"
 fi
 
-if which curl >/dev/null 2>&1 ; then
+if [ -x /usr/bin/curl ]; then
 	DL="curl -o $DEST.new $SRC"
     ${quiet} && DL="$DL -s -S"
-elif which wget >/dev/null 2>&1 ; then
+elif [ -x /usr/bin/wget ]; then
 	DL="wget -O $DEST.new $SRC"
 	${quiet} && DL="$DL -q"
-elif which lynx >/dev/null 2>&1 ; then
+elif [ -x /usr/bin/lynx ]; then
 	DL="eval lynx -source $SRC >$DEST.new"
 else
 	echo >&2 "update-pciids: cannot find curl, wget or lynx"
