--- pciutils-2.1.8/pci.ids.serveraid	Wed Feb 21 12:31:36 2001
+++ pciutils-2.1.8/pci.ids	Wed Feb 21 12:35:21 2001
@@ -377,7 +377,8 @@
 	0020  MCA
 	0022  IBM27-82351
 	002d  Python
-	002e  ServeRAID controller
+	002e  ServeRAID-3x
+	022e  ServeRAID-4H
 	0036  Miami
 	003a  CPU to PCI Bridge
 	003e  16/4 Token ring UTP/STP controller
@@ -412,6 +413,9 @@
 	0144  Yotta Video Compositor Output
 		1014 0145 Yotta Output Controller (ytout)
 	0156  405GP PLB to PCI Bridge
+	01bd  Netfinity ServeRAID controller
+	01be  ServeRAID-4M
+	01bf  ServeRAID-4L
 	ffff  MPIC-2 interrupt controller
 1015  LSI Logic Corp of Canada
 1016  ICL Personal Systems
