--- pciutils-2.1.8/pci.ids.up	Tue Dec 12 11:58:19 2000
+++ pciutils-2.1.8/pci.ids	Tue Dec 12 11:58:22 2000
@@ -4,7 +4,7 @@
 #	Maintained by Martin Mares <pci-ids@ucw.cz>
 #	If you have any new entries, send them to the maintainer.
 #
-#	$Id$
+#	$Id$
 #
 
 # Vendors, devices and subsystems. Please keep sorted.
@@ -238,11 +238,12 @@
 100a  Phoenix Technologies
 100b  National Semiconductor Corporation
 	0001  DP83810
-	0002  87415
+	0002  87415/87560 IDE
+	000e  87560 Legacy I/O
 	000f  OHCI Compliant FireWire Controller
 	0011  National PCI System I/O
 	0012  USB Controller
-	d001  87410
+	d001  87410 IDE
 100c  Tseng Labs Inc
 	3202  ET4000/W32p rev A
 	3205  ET4000/W32p rev B
@@ -570,6 +571,10 @@
 	0001  PowerEdge Expandable RAID Controller 2/Si
 	0002  PowerEdge Expandable RAID Controller 3/Di
 	0003  PowerEdge Expandable RAID Controller 3/Si
+	0004  PowerEdge Expandable RAID Controller 3/Si
+	0005  PowerEdge Expandable RAID Controller 3/Di
+	0006  PowerEdge Expandable RAID Controller 3/Di
+	0008  PowerEdge Expandable RAID Controller 3/Di
 1029  Siemens Nixdorf IS
 102a  LSI Logic
 	0000  HYDRA
@@ -919,6 +924,7 @@
 	8009  CXD1947Q i.LINK Controller
 	8039  CXD3222 i.LINK Controller
 	8056  Rockwell HCF 56K modem
+	808a  Memory Stick Controller
 104e  Oak Technology, Inc
 	0017  OTI-64017
 	0107  OTI-107 [Spitfire]
@@ -942,6 +948,10 @@
 1053  Young Micro Systems
 1054  Hitachi, Ltd
 1055  EFAR Microsystems
+	9130  EIDE Controller
+	9460  PCI to ISA Bridge
+	9462  USB Universal Host Controller [OHCI]
+	9463  Power Management Controller [Bridge]
 1056  ICL
 # Motorola made a mistake and used 1507 instead of 1057 in some chips.
 # Please look at the 1507 entry as well when updating this.
@@ -977,6 +987,8 @@
 1058  Electronics & Telecommunications RSH
 1059  Teknor Industrial Computers Inc
 105a  Promise Technology, Inc.
+	0d30  20265
+	4d30  20267
 	4d33  20246
 	4d38  20262
 	5300  DC5300
@@ -1229,6 +1241,7 @@
 	0646  PCI0646
 	0647  PCI0647
 	0648  PCI0648
+	0649  PCI0649
 	0650  PBC0650A
 	0670  USB0670
 	0673  USB0673
@@ -1248,6 +1261,7 @@
 	036c  Bt879(??) Video Capture
 		13e9 0070  Win/TV (Video Section)
 	036e  Bt878
+		0070 13eb  WinTV/GO
 		127a 0001  Bt878 Mediastream Controller NTSC
 		127a 0002  Bt878 Mediastream Controller PAL BG
 		127a 0003  Bt878a Mediastream Controller PAL BG
@@ -1295,6 +1309,7 @@
 		1851 1851  FlyVideo'98 EZ - video
 		1852 1852  FlyVideo'98 (with FM Tuner)
 	0878  Bt878
+		0070 13eb  WinTV/GO
 		127a 0001  Bt878 Video Capture (Audio Section)
 		127a 0002  Bt878 Video Capture (Audio Section)
 		127a 0003  Bt878 Video Capture (Audio Section)
@@ -1457,19 +1472,24 @@
 		10b7 3590  TokenLink Velocity XL Adapter
 	4500  3c450 Cyclone/unknown
 	5055  3c555 Laptop Hurricane
+	6055  3c556 Laptop Tornado
+	6056  3c556B Laptop Hurricane
 	5057  3c575 [Megahertz] 10/100 LAN CardBus
 		10b7 5a57  3C575 Megahertz 10/100 LAN Cardbus PC Card
-	5157  3c575 [Megahertz] 10/100 LAN CardBus
+	5157  3CCFE575BT Cyclone CardBus
 		10b7 5b57  3C575 Megahertz 10/100 LAN Cardbus PC Card
-	5257  3CCFE575CT Cyclone CardBus
+	5257  3CCFE575CT Tornado CardBus
 	5900  3c590 10BaseT [Vortex]
 	5920  3c592 EISA 10mbps Demon/Vortex
 	5950  3c595 100BaseTX [Vortex]
 	5951  3c595 100BaseT4 [Vortex]
 	5952  3c595 100Base-MII [Vortex]
 	5970  3c597 EISA Fast Demon/Vortex
+	5b57  3c595 [Megahertz] 10/100 LAN CardBus
+		10b7 5b57  3C575 Megahertz 10/100 LAN Cardbus PC Card
 	6560  3CCFE656 Cyclone CardBus
-	6562  3CCFEM656 Cyclone CardBus
+	6562  3CCFEM656B Cyclone CardBus
+	6564  3CXFEM656C Tornado CardBus
 	7646  3cSOHO100-TX Hurricane
 	8811  Token ring
 	9000  3c900 10BaseT [Boomerang]
@@ -1504,12 +1524,15 @@
 		1028 0098  3C905B Fast Etherlink XL 10/100
 		1028 0099  3C905B Fast Etherlink XL 10/100
 		10b7 9055  3C905B Fast Etherlink XL 10/100
+	9056  3c905B-T4
 	9058  3c905B-Combo [Deluxe Etherlink XL 10/100]
 	905a  3c905B-FX [Fast Etherlink XL FX 10/100]
 	9200  3c905C-TX [Fast Etherlink]
 		10b7 1000  3C905C-TX Fast Etherlink for PC Management NIC
 	9800  3c980-TX [Fast Etherlink XL Server Adapter]
 		10b7 9800  3c980-TX Fast Etherlink XL Server Adapter
+	9805  3c980-TX 10/100baseTX NIC [Python-T]
+		10b7 9805  3c980 10/100baseTX NIC [Python-T]
 10b8  Standard Microsystems Corp [SMC]
 	0005  83C170QF
 		1055 e000  LANEPIC
@@ -1745,11 +1768,13 @@
 		1043 400a  AGP-V6800 DDR SGRAM
 		1043 400b  AGP-V6800 DDR SDRAM
 		1102 102e  CT6971 GeForce 256 DDR
+		14af 5021  3D Prophet DDR-DVI
 	0103  Quadro (GeForce 256 GL)
 	0110  NV11
 	0111  NV11 DDR
 	0113  NV11 GL
 	0150  NV15 (Geforce2 GTS)
+		107d 2840  WinFast GeForce2 GTS with TV output
 	0151  NV15 DDR (Geforce2 GTS)
 	0152  NV15 Bladerunner (Geforce2 GTS)
 	0153  NV15 GL (Quadro2)
@@ -1891,35 +1916,38 @@
 1105  Sigma Designs, Inc.
 	8300  REALmagic Hollywood Plus DVD Decoder
 1106  VIA Technologies, Inc.
+	0305  VT8363/8365 [KT133/KM133]
 	0391  VT8371 [KX133]
-	0501  VT8501
+	0501  VT8501 [Apollo MVP4]
 	0505  VT82C505
 	0561  VT82C561
-	0571  VT82C586 IDE [Apollo]
+	0571  Bus Master IDE
 	0576  VT82C576 3V [Apollo Master]
 	0585  VT82C585VP [Apollo VP1/VPX]
 	0586  VT82C586/A/B PCI-to-ISA [Apollo VP]
 		1106 0000  MVP3 ISA Bridge
 	0595  VT82C595 [Apollo VP2]
-	0596  VT82C596 ISA [Apollo PRO]
+	0596  VT82C596 ISA [Mobile South]
 		1106 0000  VT82C596/A/B PCI to ISA Bridge
 		1458 0596  VT82C596/A/B PCI to ISA Bridge
 	0597  VT82C597 [Apollo VP3]
 	0598  VT82C598 [Apollo MVP3]
-	0601  VT8601
+	0601  VT8601 [Apollo ProMedia]
+	0605  VT8605 [ProSavage PM133]
 	0680  VT82C680 [Apollo P6]
-	0686  VT82C686 [Apollo Super]
+	0686  VT82C686 [Apollo Super South]
 		1106 0000  VT82C686/A PCI to ISA Bridge
 		1106 0686  VT82C686/A PCI to ISA Bridge
-	0691  VT82C691 [Apollo PRO]
+	0691  VT82C693A/694x [Apollo PRO133x]
 		1458 0691  VT82C691 Apollo Pro System Controller
+	0698  VT82C693A [Apollo Pro133 AGP]
 	0693  VT82C693 [Apollo Pro Plus]
 	0926  VT82C926 [Amazon]
 	1000  VT82C570MV
 	1106  VT82C570MV
 	1571  VT82C416MV
 	1595  VT82C595/97 [Apollo VP2/97]
-	3038  VT82C586B USB
+	3038  UHCI USB
 		1234 0925  MVP3 USB Controller
 	3040  VT82C586B ACPI
 	3043  VT86C100A [Rhine 10/100]
@@ -1927,19 +1955,31 @@
 		1106 0100  VT86C100A Fast Ethernet Adapter
 		1186 1400  DFE-530TX
 	3044  OHCI Compliant IEEE 1394 Host Controller
+	3050  VT82C596 Power Management
+	3051  VT82C596 Power Management
 	3057  VT82C686 [Apollo Super ACPI]
-	3058  VT82C686 [Apollo Super AC97/Audio]
+	3058  AC97 Audio Controller
 		1462 3091  MS-6309 Onboard Audio
-	3068  VT82C686 [Apollo Super AC97/Modem]
+	3059  AC97 Audio Controller
+	3065  Ethernet Controller
+	3068  AC97 Modem Controller
+	3074  VT8233 PCI to ISA Bridge
+	3091  VT8633 [Apollo Pro266]
+	3099  VT8367 [KT266]
 	5030  VT82C596 ACPI [Apollo PRO]
 	6100  VT85C100A [Rhine II]
 	8231  VT8231 [PCI-to-ISA Bridge]
-	8391  VT8371 [PCI-PCI Bridge]
-	8501  VT8501
+	8235  VT8235 Power Management
+	8305  VT8363/8365 [KT133/KM133 AGP]
+	8391  VT8371 [KX133 AGP] 
+	8501  VT8501 [Apollo MVP4 AGP]
 	8596  VT82C596 [Apollo PRO AGP]
 	8597  VT82C597 [Apollo VP3 AGP]
-	8598  VT82C598 [Apollo MVP3 AGP]
-	8601  VT8601
+	8598  VT82C598/694x [Apollo MVP3/Pro133x AGP]
+	8601  VT8601 [Apollo ProMedia AGP]
+	8605  VT8605 [PM133 AGP]
+	B091  VT8633 [Apollo Pro266 AGP]
+	B099  VT8367 [KT266 AGP]
 	8691  VT82C691 [Apollo Pro]
 1107  Stratus Computers
 	0576  VIA VT82C570MV [Apollo] (Wrong vendor ID!)
@@ -2045,10 +2085,18 @@
 111a  Efficient Networks, Inc
 	0000  155P-MF1 (FPGA)
 	0002  155P-MF1 (ASIC)
-	0003  ENI-25P ATM Adapter
+	0003  ENI-25P ATM
 		111a 0000  ENI-25p Miniport ATM Adapter
-	0005  Speedstream 30xx ATM Adapter
-		111a 0001  SS-3010 Miniport ATM Adapter
+	0005  SpeedStream (LANAI)
+		111a 0001  ENI-3010 ATM
+		111a 0101  ENI-3010 ATM
+		111a 0009  ENI-3060 ADSL (VPI=0)
+		111a 0109  ENI-3060CO ADSL (VPI=0)
+		111a 0809  ENI-3060 ADSL (VPI=0 or 8)
+		111a 0909  ENI-3060CO ADSL (VPI=0 or 8)
+		111a 0a09  ENI-3060 ADSL (VPI=<0..15>)
+	0007  SpeedStream ADSL 
+		111a 1001  ENI-3061 ADSL [ASIC]
 111b  Teledyne Electronic Systems
 111c  Tricord Systems Inc.
 	0001  Powerbis Bridge
@@ -2108,7 +2156,7 @@
 		1133 e004  DIVA 2.0 U
 	e010  DIVA Server BRI-2M
 		1133 e010  DIVA Server BRI-2M
-	e014  DIVA Server PRO-30M
+	e014  DIVA Server PRI-30M
 		1133 e014  DIVA Server PRI-30M
 1134  Mercury Computer Systems
 	0001  Raceway Bridge
@@ -2270,10 +2318,14 @@
 1164  Advanced Peripherals Technologies
 1165  Imagraph Corporation
 	0001  Motion TPEG Recorder/Player with audio
-1166  Relience Computer
+1166  ServerWorks
 	0007  CNB20-LE CPU to PCI Bridge
 	0008  CNB20HE
-	0009  CNB20HE
+	0009  CNB20LE
+	0010  CIOB30
+	0011  CMIC-HE
+	0200  OSB4
+	0201  CSB5
 1167  Mutoh Industries Inc
 1168  Thine Electronics Inc
 1169  Centre for Development of Advanced Computing
@@ -2413,7 +2465,7 @@
 		11ad f003  LNE100TX
 		11ad ffff  LNE100TX
 		1385 f004  FA310TX
-	c115  LNE100TX Fast Ethernet Adapter
+	c115  LNE100TX [Linksys EtherFast 10/100]
 11ae  Aztech System Ltd
 11af  Avid Technology Inc.
 11b0  V3 Semiconductor Inc.
@@ -2647,6 +2699,10 @@
 	0105  Cyclom_8Y above first megabyte
 	0200  Cyclom_Z below first megabyte
 	0201  Cyclom_Z above first megabyte
+	0300  PC300 RX 2
+	0301  PC300 RX 1
+	0310  PC300 TE 2
+	0311  PC300 TE 1
 120f  Essential Communications
 	0001  Roadrunner serial HIPPI
 1210  Hyperparallel Technologies
@@ -2705,6 +2761,7 @@
 		121a 0060  Voodoo3 3500 TV (NTSC)
 		121a 0061  Voodoo3 3500 TV (PAL)
 		121a 0062  Voodoo3 3500 TV (SECAM)
+	0009  Voodoo 4
 121b  Advanced Telecommunications Modules
 121c  Nippon Texaco., Ltd
 121d  Lippert Automationstechnik GmbH
@@ -2714,7 +2771,17 @@
 	1220  AMCC 5933 TMS320C80 DSP/Imaging board
 1221  Contec Co., Ltd
 1222  Ancor Communications, Inc.
-1223  Heurikon/Computer Products
+1223  Artesyn Communication Products
+	0003  PM/Link
+	0004  PM/T1
+	0005  PM/E1
+	0008  PM/SLS
+	0009  BajaSpan Resource Target
+	000a  BajaSpan Section 0
+	000b  BajaSpan Section 1
+	000c  BajaSpan Section 2
+	000d  BajaSpan Section 3
+	000e  PM/PPC
 1224  Interactive Images
 1225  Power I/O, Inc.
 1227  Tech-Source
@@ -2931,7 +2998,10 @@
 1277  Comstream
 1278  Transtech Parallel Systems Ltd.
 1279  Transmeta Corporation
-	0295  Virtual Northbridge
+	0295  Northbridge
+	0395  LongRun Northbridge
+	0396  SDRAM controller
+	0397  BIOS scratchpad
 127a  Rockwell International
 	1002  HCF 56k V90 FaxModem
 		127a 1002  HCF 56k V90 Modem
@@ -4504,15 +4574,22 @@
 		11d4 5340  SoundMAX Integrated Digital Audio
 	2416  82801AA AC'97 Modem
 	2418  82801AA PCI Bridge
-	2420  82801AB Chipset ISA Bridge (LPC)
-	2421  82801AB Chipset IDE
-	2422  82801AB Chipset USB
-	2423  82801AB Chipset SMBus
+	2420  82801AB ISA Bridge (LPC)
+	2421  82801AB IDE
+	2422  82801AB USB
+	2423  82801AB SMBus
 	2425  82801AB AC'97 Audio
 		11d4 0040  SoundMAX Integrated Digital Audio
 		11d4 0048  SoundMAX Integrated Digital Audio
 	2426  82801AB AC'97 Modem
 	2428  82801AB PCI Bridge
+	2440  82820 820 (Camino 2) Chipset ISA Bridge (ICH2)
+	2442  82820 820 (Camino 2) Chipset USB (Hub A)
+	2443  82820 820 (Camino 2) Chipset SMBus
+	2444  82820 820 (Camino 2) Chipset USB (Hub B)
+	2449  82820 820 (Camino 2) Chipset Ethernet
+	244b  82820 820 (Camino 2) Chipset IDE U100
+	244e  82820 820 (Camino 2) Chipset PCI
 	2500  82820 820 (Camino) Chipset Host Bridge (MCH)
 		1043 801c  P3C-2000 system chipset
 	2501  82820 820 (Camino) Chipset Host Bridge (MCH)
@@ -4569,7 +4646,12 @@
 	84c4  450KX/GX [Orion] - 82454KX/GX PCI bridge
 	84c5  450KX/GX [Orion] - 82453KX/GX Memory controller
 	84ca  450NX - 82451NX Memory & I/O Controller
-	84cb  450NX - 82454NX PCI Expander Bridge
+	84cb  450NX - 82454NX/84460GX PCI Expander Bridge
+	84e0  460GX - 84460GX System Address Controller (SAC)
+	84e1  460GX - 84460GX System Data Controller (SDC)
+	84e2  460GX - 84460GX AGP Bridge (GXB)
+	84e3  460GX - 84460GX Memory Address Controller (MAC)
+	84e4  460GX - 84460GX Memory Data Controller (MDC)
 	ffff  450NX/GX [Orion] - 82453KX/GX Memory controller [BUG]
 8800  Trigem Computer Inc.
 	2008  Video assistent component
