--- pciutils-2.1.10/Makefile~	Wed Dec 18 23:41:44 2002
+++ pciutils-2.1.10/Makefile	Wed Dec 18 23:41:44 2002
@@ -63,14 +63,15 @@
 # -c is ignored on Linux, but required on FreeBSD
 	$(DIRINSTALL) -m 755 $(ROOT)/sbin $(PREFIX)/share $(MANDIR)/man8
 	$(INSTALL) -c -m 755 -s lspci setpci $(ROOT)/sbin
-	if [ ! -f $(PREFIX)/share/pci.ids -o pci.ids -nt $(PREFIX)/share/pci.ids ] ; then \
+	if [ ! -f $(ROOT)/etc/pci.ids -o pci.ids -nt $(ROOT)/etc/pci.ids ] ; then \
 		$(INSTALL) -c -m 644 pci.ids $(PREFIX)/share ; \
-	elif [ -f $(PREFIX)/share/pci.ids ] ; then \
-		echo "$(PREFIX)/share/pci.ids is same or newer than the version to be installed, skipping." ; \
+	elif [ -f $(ROOT)/etc/pci.ids ] ; then \
+		echo "$(ROOT)/etc/pci.ids is same or newer than the version to be installed, skipping." ; \
 	fi
 	$(INSTALL) -c -m 644 lspci.8 setpci.8 $(MANDIR)/man8
 # Remove relics from old versions
-	rm -f $(ROOT)/etc/pci.ids
+	rm -f $(PREFIX)/share/pci.ids
+	rm -f $(PREFIX)/share/misc/pci.ids
 
 install-devel: all
 	install -d -m 755 $(DESTDIR)$(includedir)/pci $(DESTDIR)$(libdir)
@@ -79,7 +80,7 @@
 
 uninstall: all
 	rm -f $(ROOT)/sbin/lspci $(ROOT)/sbin/setpci
-	rm -f $(PREFIX)/share/pci.ids
+	rm -f (ROOT)/etc/pci.ids
 	rm -f $(PREFIX)/man/man8/lspci.8 $(PREFIX)/man/man8/setpci.8
 
 update-ids:
