--- pciutils-2.1.8/pci.ids.ati	Fri Jul 14 12:07:41 2000
+++ pciutils-2.1.8/pci.ids	Fri Jul 14 12:23:57 2000
@@ -140,16 +140,19 @@
 		0e11 b10e  3D Rage LT Pro (Compaq Armada 1750)
 		1028 0085  Rage 3D LT Pro
 	4c44  3D Rage LT Pro AGP-66
-	4c46  Mobility M3 AGP 2x
+	4c45  Rage Mobility M3 AGP
+	4c46  Rage Mobility M3 AGP 2x
 	4c47  3D Rage LT-G 215LG
 	4c49  3D Rage LT Pro
-	4c4d  3D Rage P/M Mobility AGP 2x
-	4c4e  3D Rage L Mobility AGP 2x
+	4c4d  Rage Mobility P/M AGP 2x
+	4c4e  Rage Mobility L AGP 2x
 	4c50  3D Rage LT Pro
 	4c51  3D Rage LT Pro
-	4c52  3D Rage P/M Mobility
-	4c53  3D Rage L Mobility
+	4c52  Rage Mobility P/M
+	4c53  Rage Mobility L
 	4c54  264LT [Mach64 LT]
+	4d46  Rage Mobility M4 AGP
+	4d4c  Rage Mobility M4 AGP
 	5041  Rage 128 PA
 	5042  Rage 128 PB
 	5043  Rage 128 PC
