--- pciutils-2.1.8/pci.ids.e1000	Mon May  7 11:13:40 2001
+++ pciutils-2.1.8/pci.ids	Mon May  7 11:14:31 2001
@@ -4509,6 +4509,8 @@
 		0e11 b123  NC1634 Gigabit Ethernet Adapter
 		1014 0119  Netfinity Gigabit Ethernet SX Adapter
 		8086 1000  EtherExpress PRO/1000 Gigabit Server Adapter
+	1001  82543 Gigabit Ethernet Adapter
+	1004  82543GC Gigabit Ethernet Adapter
 	1030  82559 InBusiness 10/100
 	1130  82815 815 Chipset Host Bridge and Memory Controller Hub
 	1132  82815 CGC [Chipset Graphics Controller] 
