diff -Nru pciutils-2.1.8/lib/configure pciutils-2.1.8.new/lib/configure
--- pciutils-2.1.8/lib/configure	Fri Apr 21 13:32:26 2000
+++ pciutils-2.1.8.new/lib/configure	Sun May 21 00:11:53 2000
@@ -80,7 +80,7 @@
 if [ -z "$ok" ] ; then
 	echo "WARNING: No real configuration access method is available."
 fi
-echo >>$c "#define PATH_PCI_IDS \"$prefix/share/pci.ids\""
+echo >>$c "#define PATH_PCI_IDS \"/etc/pci.ids\""
 if [ -f header.h ] ; then
 	echo >>$c '#define HAVE_OWN_HEADER_H'
 fi
